src/status_im/contexts/wallet/wallet_connect/events/session_responses.cljs

```diff
@@ -112,7 +125,39 @@
            [:dispatch
             [:toasts/upsert
              {:type :negative
-              :text (i18n/label :t/something-went-wrong)}]]]})))
+              :text (i18n/label :t/something-went-wrong)}]]
+           [:dispatch
+            [:centralized-metrics/track-event "dapp-sign"
+             {:dapp_name dapp-name
+              :networks  networks
+              :result    :fail}]]]})))
+
+(rf/reg-event-fx
+ :wallet-connect/respond-send-transaction-error
+ (fn [{:keys [db]}]
+   (let [[dapp-name networks] (data-store/get-dapp-name-and-networks db)]
+     (log/error "Failed to send transaction to Wallet Connect")
+     {:fx [[:dispatch [:wallet-connect/dismiss-request-modal]]
+           [:dispatch
+            [:toasts/upsert
+             {:type :negative
+              :text (i18n/label :t/something-went-wrong)}]]
+           [:dispatch
+            [:centralized-metrics/track-event "dapp-send"
+             {:dapp_name dapp-name
+              :networks  networks
+              :result    :fail}]]]})))
+
+(rf/reg-event-fx :wallet-connect/respond-send-transaction-success
+ (fn [{:keys [db]} [result]]
+   (let [[dapp-name networks] (data-store/get-dapp-name-and-networks db)]
+     (log/info "Successfully sent transaction request to Wallet connect")
```

Comment:
This log is redundant because we already log at the debug level all dispatched events. Same idea for some other events in this PR.